### ISSUE TITLE
[Feature Request] Set the default value for empty keys to 0 (Boolean)

### ISSUE THREAD

**Issue by Mouli04** [toml-lang/toml#1039]
Currently, I have to explicitly set the value to `false`, as shown below:

`
# config.toml
log = false
`

I would like the default value for an empty key to be set to `false` (Boolean), like this:

`
# config.toml
log = 
`

**Comment by arp242** [comment]
This is impossible as the type is determined by the value.

It's also just ugly and confusing; nothing works like this.

**Comment by rgmz** [comment]
> **its better** if empty key value is false or 0 (Boolean)

Even if it were possible, I can think of many instances where this wouldn't be _better_. This is something you'd implement in your application, not the TOML specification.

**Comment by pradyunsg** [comment]
> I would like the default value for an empty key to be set to `false` (Boolean), like this:

I don't think this is a good idea. As @rgmz notes, this is something that should be done at the application level; not at the TOML format level.
